addons/analytic/static/src/components/analytic_distribution/analytic_distribution.scss

```diff
@@ -5,6 +5,16 @@
         width: 20px;
     }
 
+    .analytic_distribution_centerer {
+        height: 1.5em;
+        width: 50%;
+    }
+
+    .o_field_tags {
+        height: 1.5em;
+        width: 50%;
+    }
+
```

Comment:
We don't need all of this, you just need to make the Analytic distribution field take half the width it's currently taking :)

Reply:
Also please add some more information about the changes in the commit message